server/app/controllers/applicant/ApplicantProgramsController.java

```diff
@@ -113,7 +118,8 @@ public CompletionStage<Result> index(Request request) {
   }
 
   @Secure
-  public CompletionStage<Result> view(Request request, long applicantId, long programId) {
+  public CompletionStage<Result> viewWithApplicantId(
+      Request request, long applicantId, long programId) {
```

Comment:
**nitpick:** let's take this opportunity to rename this to `show` to fit the naming schema: https://github.com/civiform/civiform/wiki/Development-standards#html-routing-convention

Reply:
Done